.changes/next-release/Bug Fix-f1a5afac-efe0-41aa-8673-60897eb82e5e.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Fixed the majority of 'security token expired' errors when using CodeWhisperer in Cloud9"
```

don't need "fixed" in the message because it will be labeled as a fix in the changelog. Just a description of the problem is needed.
```suggestion
	"description": "'security token expired' errors when using CodeWhisperer in Cloud9 with managed credentials"
```